packages/@aws-cdk/aws-servicecatalogappregistry-alpha/test/application-associator.test.ts

```diff
@@ -237,7 +237,7 @@ describe('Scope based Associations with Application with Cross Region/Account',
     const crossRegionStack = new cdk.Stack(app, 'crossRegionStack', {
       env: { account: 'account', region: 'region' },
     });
-    Annotations.fromStack(crossRegionStack).hasWarning('*', 'Environment agnostic stack determined, AppRegistry association might not work as expected in case you deploy cross-region or cross-account stack.');
+    Annotations.fromStack(crossRegionStack).hasWarning('*', 'SCAppRegistry:EnvironmentAgnosticStack: Environment agnostic stack determined, AppRegistry association might not work as expected in case you deploy cross-region or cross-account stack.');
```

Comment:
Bikeshed:

```suggestion
    Annotations.fromStack(crossRegionStack).hasWarning('*', '[SCAppRegistry:EnvironmentAgnosticStack] Environment agnostic stack determined, AppRegistry association might not work as expected in case you deploy cross-region or cross-account stack.');
```

The `[flag] What the hell` feels more natural to me for some reason... ?

Reply:
Or `What the hell [flag]`, or perhaps `What the hell [acknowledge=flag]`

Reply:
Good point, I kind of like `What the hell [ack: flag]` because it makes it more clear to the user what they need to provide to ack.